gradle/mixins/dependencies.gradle

```diff
@@ -35,5 +35,13 @@ dependencyManagement {
             entry 'slf4j-api'
             entry 'slf4j-log4j12'
         }
+        dependency group: 'org.apache.spark', name: 'spark-core_2.11', version: sparkVersion
+        dependency group: 'org.apache.spark', name: 'spark-sql_2.11', version: sparkVersion
+        dependency group: 'org.apache.spark', name: 'spark-mllib_2.11', version: sparkVersion
+        dependency group: 'org.scala-lang', name: 'scala-library', version: scalaVersion
+        dependency group: 'ai.h2o', name: 'sparkling-water-scoring_2.11', version: sparklingWaterVersion
+        dependency group: 'ai.h2o', name: 'sparkling-water-core_2.11', version: sparklingWaterVersion
```

Comment:
Do you need to have sparkling-water-core?

Reply:
I feel like we did, but not sure, can check

Reply:
ya tested without sparkling water core. no issues. removed.